i18n/states/PK.php

```diff
@@ -0,0 +1,21 @@
+<?php
+/**
+ * Pakistan's states
+ *
+ * @author      WooThemes
+ * @category    i18n
+ * @package     WooCommerce/i18n
+ * @version     2.0.0
```

Comment:
Thank You for your guidance @justinshreve . i am contributing to WooCommerce for the first time.